### ISSUE TITLE
Display Customer Name on Order Summary in Orders Page

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#4378]
## Description
- As an Enterprise User
- on the Orders page https://openfoodnetwork.org.au/admin/orders
- I want to be able to see the Customer name without opening the order for editing. Often the email is not that useful for identifying the customer, and it is time-consuming to have to open orders to see who they are

See discussion leading to this proposed solution here https://community.openfoodnetwork.org/t/enable-to-identify-customer-on-orders-view/1482/21

## Acceptance Criteria & Tests

As an Enterprise user managing orders, I can see the customer name for each order from Orders page, without having to open them to edit

Suggested position under the email. 
![image.png](https://images.zenhubusercontent.com/54b83fb37c6ccc6277a6ea46/ef8df8b9-7bee-4d39-8b5f-d985c74651b5)

The email and name text could/should be centrally vertically aligned within the row, same as other fields

**Comment by chaserx** [comment]
Hi. @kirstenalarsen I'm new here. Is there an easy way to seed an order?

**Comment by chaserx** [comment]
Nevermind. I figured it out. PR submitted.

**Comment by luisramos0** [comment]
good point, we dont have a way to seed orders! Our sample data does not include orders. I think we should add orders in different states! I created an issue for it:
https://github.com/openfoodfoundation/openfoodnetwork/issues/4388

**Comment by kirstenalarsen** [comment]
Thanks @chaserx !!

**Comment by chaserx** [comment]
you're welcome! happy to help @kirstenalarsen
